pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -203,7 +203,7 @@ func validateSystemRequirements(mountUtil mount.Interface) (features, error) {
 // TODO(vmarmol): Add limits to the system containers.
 // Takes the absolute name of the specified containers.
 // Empty container name disables use of the specified container.
-func NewContainerManager(mountUtil mount.Interface, cadvisorInterface cadvisor.Interface, nodeConfig NodeConfig, failSwapOn bool, recorder record.EventRecorder, kubeClient clientset.Interface) (ContainerManager, error) {
+func NewContainerManager(ctx context.Context, mountUtil mount.Interface, cadvisorInterface cadvisor.Interface, nodeConfig NodeConfig, failSwapOn bool, recorder record.EventRecorder, kubeClient clientset.Interface) (ContainerManager, error) {
```

Comment:
Still see references to `klog.InfoS` in this function, think we should transition to  `logger.Info` as per https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#contextual-logging-in-kubernetes.

Reply:
I guess this is because this code doesn't belong to cpumanager. It should be changed when migrating container manager.

Reply:
@bart0sh exactly. I'm changing this file anyway because I need to pass down the context. The minimal change would be to still change this function and pass down a `context.TODO()`. I'll review and update the PR ASAP.

I can help migrating `k/k/pkg/kubelet/cm` later on if the effort isn't started already

Reply:
Yep, this is exactly the approach I used in https://github.com/kubernetes/kubernetes/pull/129942 and outlined in https://github.com/kubernetes/kubernetes/issues/130069. I think it makes sense to continue with it.

Reply:
Thanks for clarifying, makes sense!